models/contrib/shivam/uniswap_var_1.py

```diff
@@ -0,0 +1,179 @@
+from credmark.cmf.model import Model
```

Comment:
@nishgaba-ai @leafyoung Let's take a look at which parts of this are reusable for the future. otherwise it's a good example of using dataframes and fully expanded dtos.